build.rs

```diff
@@ -93,10 +93,31 @@ impl BuildConstants {
     }
 }
 
+fn git_short_hash() -> String {
+    let output = Command::new("git")
+        .args(["rev-parse", "--short", "HEAD"])
+        .output();
+
+    if let Ok(output) = output {
```

Comment:
When would this fail?

Reply:
The `output()` call actually executes the command in a subprocess so there could be a lot of reasons it could fail I guess. Since this is in a critical code path for compilation I figured this would be safer than doing an `unwrap()` .

But I'm not _expecting_ it to fail. I could add a clarifying comment if that would be helpful.